upload.sh

```diff
@@ -10,13 +10,17 @@ if [ ! -z "$UPLOAD_URL" ]; then
     ls $SRC_URL/*HF*.zip >/dev/null 2>&1 && exit 0 || true
     date
     scp -C $SRC_URL/*.zip* $UPLOAD_URL || true
-    mkdir -p $HERE/download/mp
-    cd $HERE/download/mp
-    links=`lynx --dump $LASTBUILD_URL/mp | grep -E -o 'https?:.*archives\/((nuxeo-.*(-sdk)*.zip(.md5)*)|packages.xml)' | sort -u`
+    MP_DIR=$HERE/download/mp-nuxeo-server
+    mkdir -p $MP_DIR
+    cd $MP_DIR
+    links=`lynx --dump $LASTBUILD_URL/mp-nuxeo-server -listonly grep -E -o 'https?:.*archives\/mp[a-z-]+\/([a-z-]+\.zip|packages.xml|\.featured)' | sort -u`
```

Comment:
Missing `|` before `grep`

Reply:
`[a-z-]+` seems a bit restrictive.

Reply:
yes, it was covering the current naming; considering the fact that this download method is deprecated and to be replaced with the Jenkins copy artifact feature.
I wanted to filter out `/*zip*/` from this output:

```
https://qa.nuxeo.org/jenkins/job/Deploy/job/IT-nuxeo-master-build/lastSuccessfulBuild/artifact/archives/mp-nuxeo-server/nuxeo-vision.zip
https://qa.nuxeo.org/jenkins/job/Deploy/job/IT-nuxeo-master-build/lastSuccessfulBuild/artifact/archives/mp-nuxeo-server/packages.xml
https://qa.nuxeo.org/jenkins/job/Deploy/job/IT-nuxeo-master-build/lastSuccessfulBuild/artifact/archives/mp-nuxeo-server/*zip*/mp-nuxeo-server.zip
```